File: ankiety-frontend/src/app/models/survey.ts
import { Question } from "./question";

export class Survey {

  constructor() {
    this.questions = new Array<Question>();
  }

  id: string;
  name: string;
  description: string;
  type: string;
  sent: boolean;
  sentInvitations: number;
  filledInvitations: number;
  questions: Question[];
}

File: ankiety-backend/Surveys.Core/Entities/Survey.cs
using System;
using Surveys.Core.Enums;
using System.Collections.Generic;

namespace Surveys.Core.Entities
{
    public class Survey
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public SurveyType SurveyType { get; set; }

        public DateTime Created { get; set; }

        public int? QuestionsOnPage { get; set; }

        public virtual ICollection<Question> Questions { get; set; }
    }
}
